sound/soc/sof/pcm.c

```diff
@@ -298,11 +298,11 @@ static int sof_pcm_trigger(struct snd_soc_component *component,
 
 	switch (cmd) {
 	case SNDRV_PCM_TRIGGER_PAUSE_PUSH:
-		ipc_first = true;
 		break;
 	case SNDRV_PCM_TRIGGER_PAUSE_RELEASE:
 		break;
 	case SNDRV_PCM_TRIGGER_START:
+		ipc_first = true;
```

Comment:
do we need this for PAUSE_RELEASE?

Reply:
Yes, better to keep it aligned with START